Package.swift

```diff
@@ -20,7 +20,7 @@ let package = Package(
         .package(url: "https://github.com/SnapKit/SnapKit.git", exact: "5.6.0"),
         .package(url: "https://github.com/thumbtack/TTCalendarPicker.git", exact: "0.2.0"),
         .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", exact: "1.16.0"),
-        .package(url: "https://github.com/thumbtack/thumbprint-tokens.git", exact: "13.0.2"),
```

Comment:
i see you didn't update the package.swift file to switch to using strict concurrency. Was that an intentional decision ?

Reply:
yeah we could do that independently from switching to strict concurrency

Reply:
i'm not sure i follow ? You did update the xcodeproj but not the package.swift, so the thumbprint target won't use targeted or complete concurrency. 

Are you just saying we should make that change separately ?

Reply:
we will do the strict concurrency for framework separately as there is a lot of warnings when we turn it on